pagopa/io-app
# feat: [IOCOM-571] Remove `pnNoticesF24Enabled` feature flag

## Short description
This PR removes the `pnNoticesF24Enabled` feature flag and all the related references also removes the pn prefix from all components.

## List of changes proposed in this pull request
- removed feature flag
- removed legacy components
- removed the pn prefix from all components

## How to test
Using `io-dev-api-server`, navigate to the detail of a PN message and check that everything is working properly
